### PR TITLE
Add documentation for Playbooks

### PR DESCRIPTION
**Pull request by adityathebe**
Resolves: https://github.com/flanksource/docs/issues/88

### CODE REVIEW

File: docs/playbooks/concepts/playbook/examples.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Examples
+
+## 1. Scaling EC2 instance
+
+```yaml
+description: Scale deployment
```
Comment (moshloop) [inline docs/playbooks/concepts/playbook/examples.md:6]: Can we add in the CRD metadata

File: docs/playbooks/concepts/playbook/examples.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      namespace: default
+      cluster: local-kind-cluster
+parameters:
+  - name: replicas
+    label: The new desired number of replicas.
+approval:
+  type: any
+  approvers:
+    people:
+      - d87243c9-3183-4ab9-9df9-c77c8278df11
```
Comment (moshloop) [inline docs/playbooks/concepts/playbook/examples.md:19]: These should be names or emails - not ids

File: docs/playbooks/concepts/playbook/examples.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    teams:
+      - 018770c4-4b73-5d44-8bb5-0e849d62e461
+actions:
+  - name: 'scale deployment'
+    exec:
+      script: kubectl scale --replicas={{.params.replicas}} deployment {{.config.name}}
```
Comment (moshloop) [inline docs/playbooks/concepts/playbook/examples.md:25]: I think we are missing the namespace of the config

File: docs/playbooks/concepts/approval/approve-run.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Approving a run
```
Comment (moshloop) [inline docs/playbooks/concepts/approval/approve-run.md:1]: I would merge all the API calls into a single API file

File: docs/playbooks/concepts/run/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Playbook Run
```
Comment (moshloop) [inline docs/playbooks/concepts/run/index.md:1]: merge this into the main index.md, with statues going after the main spec table

File: docs/playbooks/quick-start.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+In this walkthrough, we'll create and run a playbook that scales a Kubernetes deployment.
+The only prerequisites are
+
+- you have a config item for a Kubernetes Deployment
+- you have kubectl configured.
+
+## 1. Create the playbook
+
+```yaml
```
Comment (moshloop) [inline docs/playbooks/quick-start.md:11]:
```suggestion
```yaml title="scale-deployment.yaml"
```

File: docs/playbooks/quick-start.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  parameters:
+    - name: replicas
+      label: The new desired number of replicas.
+  actions:
+    - name: 'scale deployment'
+      exec:
+        script: kubectl scale --replicas={{.params.replicas}} deployment {{.config.name}}
+```
+
+This playbook is designed to run a configs of type `Kubernetes::Deployment` & having labels namespace: `default` & cluster: `local-kind-cluster`.
```
Comment (moshloop) [inline docs/playbooks/quick-start.md:32]:
```suggestion
This playbook is designed to run on a Kubernetes Depolyment (with a config type of `Kubernetes::Deployment`)
```

File: docs/playbooks/quick-start.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+kubectl apply -f scale-deployment.yaml
+```
+
+Please check the `playbooks` table in the database to see if the playbook has been created.
+
+## 2. Run the playbook
+
+To run the playbook, we need its id. In addition to that please get the id for the deployment config item that you want to scale.
+
+```bash
```
Comment (moshloop) [inline docs/playbooks/quick-start.md:47]: Rather use CRD definition to apply

File: docs/playbooks/concepts/approval/index.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# Approval
```
Comment (moshloop) [inline docs/playbooks/concepts/approval/index.md:1]: merge into main index.md

File: docs/playbooks/concepts/playbook/examples.md (hunk 2 above)
Comment (adityathebe) [reply docs/playbooks/concepts/playbook/examples.md:19]: @moshloop will make this email only. name is not unique.

File: docs/playbooks/quick-start.md (hunk 8 above)
Comment (adityathebe) [reply docs/playbooks/quick-start.md:47]: Wdym?

File: docs/playbooks/concepts/run/index.md (hunk 5 above)
Comment (adityathebe) [reply docs/playbooks/concepts/run/index.md:1]: I think it warrants its own page. (Same with approval)

**Pull request merged**